crates/re_analytics/src/lib.rs

```diff
@@ -152,30 +203,31 @@ impl Analytics {
         let sink = PostHogSink::default();
         let pipeline = Pipeline::new(&config, tick, sink)?;
 
-        if let Some(pipeline) = pipeline.as_ref() {
-            pipeline.record(Event::update_metadata(config.metadata.clone()));
-        }
-
         Ok(Self {
             config,
             default_append_props: Default::default(),
             pipeline,
-            event_id: AtomicU64::new(1),
+            event_id: AtomicU64::new(0),
```

Comment:
I'm not sure if there was some reason this wasn't `0` except for the `update_metadata` being sent before this